__tests__/app.test.js

```diff
@@ -58,6 +58,15 @@ describe("ERROR Handling", () => {
         expect(body.msg).toBe("Article not found!");
       });
   });
+  test("status:400 for a request where the object is empty", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid request!");
+      });
+  });
```

Comment:
There's a few more error handling tests to add for PATCH
think about if the article id is invalid/doesn't exist
if invalid inc_votes type, e.g. property is not a number